apps/application-status/pages/index.vue

```diff
@@ -81,21 +64,54 @@ export default {
       state: '',
       totalPoints: 0,
       updatedAt: '',
-      numberOfRelease: 0
-    }
-    const entities: Response = {
-      sprint,
-      releases: []
+      numberOfRelease: 0,
+      issues: {
+        totalCount: 0
+      }
     }
-    const nameteam: Response = {
-      sprint,
-      releases: []
+    const result = {
+      totalReleases: 0,
+      totalIssues: 0,
+      totalBugs: 0
     }
+    const teams = [
+      {
+        id: 1,
+        title: 'ENTITIES',
```

Comment:
since you are already using BoardName enum, why don't you also do something similar for title attribute - these are hardcoded constants as well. You basically want to have one central place where all these constants can  be edited